README.md

```diff
@@ -18,12 +18,30 @@ single-threaded, Modin lets you instantly speed up your workflows by scaling pan
 cores. Modin works especially well on larger datasets, where pandas becomes painfully slow or runs
 [out of memory](https://modin.readthedocs.io/en/latest/getting_started/why_modin/out_of_core.html).
 
-Using modin is as simple as replacing the pandas import:
-
-```python
-# import pandas as pd
-import modin.pandas as pd
-```
+By simply replacing the import statement, Modin offers users effortless speed and scale for their pandas workflows:
+
+<img src="docs/img/Import.gif" style="display: block;margin-left: auto;margin-right: auto;" width="100%"></img>
+
+In the GIFs below, Modin (left) and pandas (right) perform *the same pandas operations* on a 2GB dataset. The only difference between the two notebook examples is the import statement. 
+
+<table class="tg">
+<thead>
+  <tr>
+    <th class="tg-0lax" style="text-align: center;"><img src="https://github.com/modin-project/modin/blob/master/docs/img/MODIN_ver2_hrz.png?raw=True" height="35px"></th>
+    <th class="tg-0lax" style="text-align: center;"><img src="https://pandas.pydata.org/static/img/pandas.svg" height="50px"></img></th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td class="tg-0lax"><img src="docs/img/Modin.gif"></img></td>
+    <td class="tg-0lax"><img src="docs/img/Pandas.gif"></img></td>
```

Comment:
For both the links to `examples/jupyter` and GIFs, is it better to use absolute links to the URL or local refs? My sense is that absolute links are better so that README still work in isolated contexts. If so, let's replace the links, which means the links will break for now until the PR is merged.